src/main/java/gov/cms/madie/resources/CustomErrorHandlerAdvice.java

```diff
@@ -31,6 +32,14 @@ Map<String, Object> handleCustomException(WebRequest request) {
     return getErrorAttributes(request, HttpStatus.BAD_REQUEST);
   }
 
+  // Handle HQMFServiceException separately
+  @ExceptionHandler(HQMFServiceException.class)
+  @ResponseStatus(HttpStatus.BAD_REQUEST)
+  @ResponseBody
+  Map<String, Object> handleHQMFServiceException(WebRequest request) {
+    return getErrorAttributes(request, HttpStatus.INTERNAL_SERVER_ERROR);
```

Comment:
lil bit confusing to me:
line 37 -> HttpStatus.BAD_REQUEST
line ->INTERNAL_SERVER_ERROR

Reply:
okay, after looking at this closely- we do not need to introduce new exception. All you have to do it to remove try catch in src/main/java/gov/cms/madie/services/PackagingService.java